PyTorch_dcgan_lsun/README.md

```diff
@@ -0,0 +1,116 @@
+# PyTorch - DCGAN
+
+This example is based on [this implementation](https://github.com/pytorch/examples/tree/master/dcgan)
+
+
+## Downloading the dataset
+
+You can download the LSUN dataset by cloning [this repo](https://github.com/fyu/lsun) and running
+```
+python download.py -c bedroom
```

Comment:
Do we have to download the whole dataset in order to run the example/benchmark?

Reply:
above command only downloads part of the dataset

Reply:
lsun (master*) $ du -sh *
32G	bedroom_train_lmdb.zip

We should not require downloading a 32GB dataset only to run a performance benchmark. In the case of our DeepSpeech example, only one training example is used.

BTW. installation of Python request package should be included in the README, maybe through requirements.txt...

Reply:
The owner doesn't provide an option to download smaller part of the dataset. Should we host a few example images then? I couldn't find any information about the license on their website - https://www.yf.io/p/lsun

Reply:
Good question... Maybe we should consider another small dataset with clear licensing?

Reply:
Well, we could use mnist then

Reply:
The DCGAN implementation used supports some datasets. One of them is "fake":

 dataset = dset.FakeData(image_size=(3, opt.imageSize, opt.imageSize),
                            transform=transforms.ToTensor())

Maybe this would work? We should check it. If it works for the benchmark, we could suggest using it in the README, and also suggest using other datasets to actucally train something.

Reply:
Added commands for running with fake dataset and updated patch to allow for it to work